{"name": "Проверка ошибки при попытке удаления курьера с несуществующим id", "status": "passed", "steps": [{"name": "Удаление курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "'234563256'"}], "start": 1747228288534, "stop": 1747228288615}], "start": 1747228288533, "stop": 1747228288617, "uuid": "61b832d1-cfd4-43f8-9bfb-941b097eddec", "historyId": "3f588d613e3f1bdea1b2b2aa5c05c8e5", "testCaseId": "3f588d613e3f1bdea1b2b2aa5c05c8e5", "fullName": "tests.test_couriers.test_courier_delete.TestCourierDelete#test_courier_delete_error_with_empty", "labels": [{"name": "parentSuite", "value": "tests.test_couriers"}, {"name": "suite", "value": "test_courier_delete"}, {"name": "subSuite", "value": "TestCourierDelete"}, {"name": "host", "value": "DESKTOP-VE6DRBG"}, {"name": "thread", "value": "19428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_couriers.test_courier_delete"}]}